resources/aro/exploring-available-images.md

```diff
@@ -0,0 +1,35 @@
+## Check #1: Exploring the images you leverage and its availability
+
+Double check your build and deployment infrastructure for the images you are leveraging. 
+There is a __high likelihood__ that some of the images you are referencing will not exist inside your new namespace, the default `openshift` namespace etc. 
+
+> This is also a good opportunity to discover out-of-date images and upgrade them wherever possible
+
+
+### S2I builds 
+
+There is a high chance some of your s2i builder images do not exist within the namespace. You will need to rebuild these inside of your `tools` namespace and update references to the s2i builder within your `BuildConfig`(s).
+
+### RedHat Images
+
+There is a high chance some of the RedHat Images you were referencing do not already exist within your namespace or the default `openshift` one. You can easily pull these using a Redhat Service Account. 
```

Comment:
This is one target where we hope to leverage Artifactory as a "Pull through" and caching service (will use a RH Service account provided by artifactory).

The following look good as an interim (until artifactory is rolled out and available.).. That said, I haven't looked at the consumption/provisioning process for azure in an on-prem artifactory service yet.